artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/etc/artemis.profile

```diff
@@ -41,6 +41,11 @@ JAVA_ARGS="${java-opts} -XX:+PrintClassHistogram -XX:+UseG1GC -Xms512M -Xmx2G -D
 # Replace "all_pauses.log" with the file name you want to log to.
 # JAVA_ARGS="$JAVA_ARGS -XX:+PrintSafepointStatistics -XX:PrintSafepointStatisticsCount=1 -XX:+PrintGCApplicationStoppedTime -XX:+PrintGCApplicationConcurrentTime -XX:+LogVMOutput -XX:LogFile=all_pauses.log"
 
+#
+# Enables the dumping of the java heap to the file oom_dump.hprof by using the heap profiler (HPROF)
+# when a java.lang.OutOfMemoryError exception is thrown.
+# JAVA_ARGS="$JAVA_ARGS -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=${ARTEMIS_INSTANCE}/log/oom_dump.hprof"
```

Comment:
Would be good to include timestamp to the heapdump file name

Reply:
I set a fixed filename to prevent the creation of multiple heap dumps that can fill the disk.

Reply:
timestamp would be generated based on when the broker started on this case.. not when the dump happened... 

I would keep it simple here.

If you want a variable name.. fine.. but other than that... you start the broker now.. it dumps 5 months from now.. the timestamp is useless.